metricbeat/metricbeat.reference.yml

```diff
@@ -1267,3 +1267,19 @@ logging.files:
 
 # Set to true to log messages in json format.
 #logging.json: false
+
+
+#================================ HTTP Endpoint ======================================
+# Each beat can expose internal data points through a http endpoint. For security
```

Comment:
- "data points" - "metrics" instead?
- Capitalize `http`.
- `s/reason/reasons/`
- Put a period after `/stats` to end the sentence.
- "For pretty JSON output append a `?pretty` query string to the URL.

Reply:
fixed